src/client.py

```diff
@@ -0,0 +1,30 @@
+"""Client for communicating with http server."""
+
+
+import socket
+import sys
+
+
+def client(message):
+    """Create the protocol for interacting with server."""
+    if len(message) % 8 == 0:
+        message += '$'
+    info = socket.getaddrinfo('127.0.0.1', 6024)
+    stream_info = [i for i in info if i[1] == socket.SOCK_STREAM][0]
+    client_socket = socket.socket(*stream_info[:3])
+    client_socket.connect(stream_info[-1])
+    buffer_length = 8
+    response = u""
+    client_socket.sendall(message.encode('utf-8'))
```

Comment:
This is *still* going to break in Python 2. You're not accounting for the fact that a string in Python2 is a bytestring by default. If a message comes through with non-standard ascii characters like ü or ñ, then you'll get an error back like this:

```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)
```